docs/components/input-field/demo/base-demo.md

```diff
@@ -4,9 +4,16 @@
   @hint='Type "input" into the field'
   @error={{this.errorMessage}}
   @value={{this.value}}
-  @onChange={{this.updateValue}}
+  @onChange={{this.handleChange}}
+  class='w-full'
   type='text'
-/>
+>
+  <:secondary as |secondary|><secondary.CharacterCount
+      @max={{255}}
+      class="type-xs-tight"
```

Comment:
If this is the styling we want for character count, can it be added on the component instead of relying on consumers to always pass this in?